typeorm/typeorm-store/src/database.ts

```diff
@@ -110,15 +122,13 @@ export class TypeormDatabase {
 
         assert(status.length == 1)
 
-        let top: HashAndHeight[] = await em.query(
-            `SELECT hash, height FROM ${schema}.hot_block ORDER BY height`
-        )
+        let top: HashAndHeight[] = await em.query(`SELECT hash, height FROM ${schema}.hot_block ORDER BY height`)
 
         return assertStateInvariants({...status[0], top})
     }
 
     transact(info: FinalTxInfo, cb: (store: Store) => Promise<void>): Promise<void> {
-        return this.submit(async em => {
+        return this.submit(async (em) => {
```

Comment:
Hey, @belopash , can you change your IDE settings to not introduce unnecessary parenthesis. Code looks cleaner without them!

Reply:
Oops, sorry, tried not to format existing files